### DISCUSSION TITLE
提个关于图片上传的建议

### DISCUSSION THREAD

**Discussion by Bytelv** [Qexo/Qexo#301] [Ideas]
大部分时候，我想上传图片到github图床，我必须打开一篇文章，然后点击上传图片。
能否在图片页面增加一个上传图片的按钮？

**Comment by am-abudu** [comment]
我建议你使用Picgo等软件实现，这会方便许多

**Reply by Bytelv** [reply]
虽然是这样，但我在日常使用时都会下意识地使用qexo上传图片。我阅读了源码，发现只要写一个fuction就可以实现这个功能，但我不知道怎样调用/api/upload/

**Reply by am-abudu** [reply]
POST请求 并在file[]里面提交二进制的图片流

**Reply by Bytelv** [reply]
好的👌我今天试一下能不能写出这个function😆
